packages/suite/src/middlewares/wallet/storageMiddleware.ts

```diff
@@ -64,19 +63,6 @@ const storageMiddleware = (api: MiddlewareAPI<Dispatch, AppState>) => {
                 }
             }
 
-            if (fiatRatesActions.updateFiatRate.match(action)) {
-                api.dispatch(storageActions.saveFiatRates());
-            }
-
-            if (fiatRatesActions.removeFiatRate.match(action)) {
-                api.dispatch(
-                    storageActions.removeFiatRate(
-                        action.payload.symbol,
-                        action.payload.tokenAddress,
-                    ),
-                );
-            }
-
```

Comment:
Why is this removed?

Reply:
also removeFiatRate in storageActions is no more used but not deleted

Reply:
We don't have `updateFiatRate` and `removeFiatRate` in `fiatRatesActions.ts` so it we can't have it there anymore, was going to ask if this is something important or not?

Will remove these actions from these and the imports (should also solve the first thing in the review)

Reply:
f0796072864f0ae73f74e1d235b43c4dc83a2dfb

Reply:
saveFiatRates still called from storageMiddleware...

Reply:
I guess it was important, it stored downloaded fiat rates persistently in storage, so user don't have to download them all again on app reload/start. Not sure if some equivalent is needed or not. But even if not, this would need storage migration - no more needed data should be removed from the storage.

Reply:
As we discussed on Friday, migration will solve this (aa141d33ce5dcf686c307a780d6d3382bee8f1ec)